### DISCUSSION TITLE
Greetings

### DISCUSSION THREAD

**Discussion by iamdanieledu** [Cyfrin/foundry-full-course-cu#1649] [General]
Hello World! I'm Daniel, an Ethical Hacker specializing in penetration testing. I've always been fascinated by blockchain technology and Smart Contracts though I'm a newbie to this field. I'm eager to learn more about how I can leverage my background in security and pentesting to contribute to the broader blockchain ecosystem. I’d really appreciate any guidance on ways I can bring my skills into play here. Looking forward to collaborating with you all and contributing to this innovative community!

**Comment by dassmod** [comment]
Wish you enjoying learning Daniel!

**Comment by EngrPips** [comment]
Welcome aboard, Daniel. I wish you the best in this new pursuit of yours.
